### DISCUSSION TITLE
The settings button disappeared in Profile page. How could I get it back?

### DISCUSSION THREAD

**Discussion by mayan8860** [dayanch96/BHTikTok-Plus#40] [Q&A]
After I changed the settings button to the Profile page, it didn't appear at top left corner, and I couldn't find it in Settings page, either.

I have deleted this tweak in Sileo and re-installed it again, but it disappeared permanently.

Could you tell me how to get the BHTikTok Plus settings back again?

TikTok v34.1.0
BHTikTok Plus v2.3

Thank you!

**Comment by MhmdSAbdlh** [comment]
I had to delete and reinstall the app to let it work
